src/app/about/page.tsx

```diff
@@ -1,54 +1,47 @@
-import Image from 'next/image';
 import React from 'react';
+import { Typography } from '@mui/material';
+import Container from '@mui/material/Container';
+import './about.scss';
 
 export default function About() {
   return (
-    <>
-      <div className="centered-elements">
-        <h1>About</h1>
-        <p className="description">Placeholder</p>
-        <br></br>
-        <h2>Our Founder</h2>
-        <Image
-          src="https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png"
-          alt="John Doe"
-          width={500}
-          height={500}
-          className="profile"
-        />
-        <p className="description">Placeholder</p>
-        <br></br>
-        <h2>Person 1</h2>
-        <Image
-          src="https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png"
-          alt="John Doe"
-          width={500}
-          height={500}
-          className="profile"
-        />
-        <p className="description">Placeholder</p>
-        <br></br>
-        <h2>Person 2</h2>
-        <Image
-          src="https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png"
-          alt="John Doe"
-          width={500}
-          height={500}
-          className="profile"
-        />
-        <p className="description">Placeholder</p>
-        <br></br>
-        <h2>Person 3</h2>
-        <Image
-          src="https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_960_720.png"
-          alt="John Doe"
-          width={500}
-          height={500}
-          className="profile"
-        />
-        <p className="description">Placeholder</p>
-        <br></br>
-      </div>
-    </>
+    <Container maxWidth="lg" className="container">
+      <Typography
+        className="about-header"
+        variant="h3"
+        sx={{
```

Comment:
Why don't you move this to the scss file?

Reply:
I will move it to scss.